src/privateDogu.html

```diff
@@ -0,0 +1,618 @@
+<!-- FIXME currently not available at all, third party dogus will need to re-evaluated in the future regarding licenses -->
```

Comment:
Could you elaborate on this `FIXME`? Is there any form of task or ticket for this because the amount of work (or what re-evaluation means) is unclear to me

Reply:
We spoke with Legal in regards to third party dogu licensing. Since there was some uncertainty regarding Licensing of third party products, dockerfiles and more. That there would be a follow-up discussion that is handled separately by Legal and Requirements Engineering Personnel. 

I added the FIXME, just so that this File isn't added back into our defined routes when it shouldn't. 

Is this clear enough? If not, let's communicate over our usual channels.